docs/content/tutorials/tutorial-add-radius/index.md

```diff
@@ -321,4 +321,5 @@ This output shows that Radius has detected the three container resources in the
 
 - To learn more about authoring Radius applications, visit the [Authoring applications guide]({{< ref "guides/author-apps" >}})
 - To learn more about deploying applications using Radius, visit the [Deploying applications guide]({{< ref "guides/deploy-apps" >}})
+- To learn more about using [Radius Connections]({{< ref "guides/author-apps/containers/overview#connections" >}}) to connect your containers and resources, visit the [Connect to dependencies guide]({{< ref "guides/author-apps/containers/howto-connect-dependencies" >}})
```

Comment:
This link doesn't cover the annotation to use when a user is trying to add connections to existing k8s manifests. I think we should add that in addition to crosslinking. May be a simplified version of [step 6 from helm tutorial](https://docs.radapp.io/tutorials/helm/#step-6-add-connection)

Reply:
Thanks for the suggestion @Reshrahim - I've updated the cross linking to specifically address the community feedback you've mentioned. However, I think there is a separate issue where the Annotations features aren't documented beyond the tutorials, so I've created this issue to track that work: https://github.com/radius-project/docs/issues/1117

Reply:
sounds good!